quickstarts/rest/Search_Grounding.ipynb

```diff
@@ -94,7 +94,7 @@
         "id": "aIn2wj3tsh4Q"
       },
       "source": [
-        "Call search grounding. The `dynamic_threshold` is optional and sets the threshold that determines whether the query could be backed by grounding. Setting to 1.0 would require 100% confidence from the model, and setting to 0.0 would invoke search grounding on any input. You can see the score in the `googleSearchDynamicRetrievalScore` response field."
+        "Call search grounding."
```

Comment:
Couple more fixes needed here:
* There's still a `dynamic_retrieval_config` reference in the curl inputs.
* The tool needs to be updated to `google_search`
* AIUI, the outputs saved in this notebook are from running with a 1.5 model, not 2.0, so we need to make sure the notebook is re-run and that the outputs saved match the model in the code (otherwise we're misrepresenting model output)

Reply:
Damn, it reset on me at some point. Updating.